projects/ui-components/lf-metadata/field-components/field-base-parts/lf-field-base/date-time-field/date-time-field.component.css

```diff
@@ -18,15 +18,65 @@
   border-radius: 24px;
 }
 
-.datetime-icon {
+/* .datetime-icon {
   font-size: 22px;
-}
+} */
 
-/* TODO: this is only a temporary fix for Angular datetime picker style issues. Remove this if there is better solution.*/
+/* TODO: this is only a temporary fix for Angular datetime picker style issues. Remove this if there is better solution. */
 ::ng-deep ngx-mat-calendar.lf-datetime-picker-overlay .mat-calendar-body-cell {
-  position: relative;
+  position: absolute;
+  height: 100%;
 }
 
 ::ng-deep ngx-mat-calendar.lf-datetime-picker-overlay .mat-calendar-arrow {
   width: 0;
 }
+
+::ng-deep .mat-datepicker-content {
+  background-color: aliceblue;
```

Comment:
this should be more specific.
Is this something that should switch color based on if it's in microsoft vs. lf? We don't have aliceblue anywhere else

Reply:
Removed the alice blue
Something I was trying during testing, good catch